app/src/main/java/com/doctor/yumyum/presentation/ui/write/WriteFragment2.kt

```diff
@@ -1,14 +1,75 @@
 package com.doctor.yumyum.presentation.ui.write
 
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.navigation.fragment.findNavController
 import com.doctor.yumyum.R
 import com.doctor.yumyum.common.base.BaseFragment
-import com.doctor.yumyum.databinding.FragmentWriteFirstBinding
 import com.doctor.yumyum.databinding.FragmentWriteSecondBinding
 
 /**
  *  레시피 작성하기 2
  *  - 레시피 이름, 메인메뉴 입력하기
  *  - 추가하는 재료 & 빼는 재료 보여주기
  */
-class WriteFragment2 : BaseFragment<FragmentWriteSecondBinding>(R.layout.fragment_write_second) {
+
+class WriteFragment2 : BaseFragment<FragmentWriteSecondBinding>(R.layout.fragment_write_second), View.OnClickListener{
+    private lateinit var changeIngredients : ActivityResultLauncher<Intent>
+
+    companion object {
+        private const val REQUEST_CODE_ADD_INGREDIENTS = 9001
+        private const val REQUEST_CODE_MINUS_INGREDIENTS = 9002
```

Comment:
```suggestion
       const val REQUEST_CODE_ADD_INGREDIENTS = 9001
       const val REQUEST_CODE_MINUS_INGREDIENTS = 9002
```
다른곳에서 request code 가져올 여지가 있는 듯 싶어요, private 는 풀어주는게 어떨까요?